randobot/handler.py

```diff
@@ -165,6 +173,19 @@ async def chat_message(self, data):
 
     async def race_data(self, data):
         await super().race_data(data)
+        if self._race_pending() and self.state.get('password_active') and not self.state['password_published']:
+            seed_password = self.state['seed_password']
+            await self.set_bot_raceinfo('File Select Password: %{seed_password} -- Hash: %(seed_hash)s\n%(seed_url)s' % {
```

Comment:
I think something shorter would be nice so the list of races doesn't become too cluttered:

```suggestion
            await self.set_bot_raceinfo('%(seed_hash)s | Password: %{seed_password}\n%(seed_url)s' % {
```

Reply:
We *could* leave the password out of the bot info entirely and just utilize the message pin feature like we did with draft. That would eliminate any clutter concerns

Reply:
that's a little risky since the pin is removed once the race starts, which i think is a good decision. Either we delay that removal by like a minute or we keep it up there.

Reply:
It actually doesn't have to be removed. We never removed the pin in season 7. There could be a way to remove it once the race has finished though